apps/web/components/sections/home/hero.tsx

```
@@ -2,111 +1,111 @@
-            duration: 2,
-            repeat: Number.POSITIVE_INFINITY,
-            ease: 'easeInOut',
-          }}
-        >
-          {category.icon}
-        </motion.span>
-        {category.label}
-      </Badge>
-    </motion.div>
-  ), []);
+	const renderCategory = useCallback(
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: shouldReduceMotion

This dependency is not specified in the hook dependency list.

Either include it or remove the dependency array

(lint/correctness/useExhaustiveDependencies)